components/cover/basic/src/index.scss

```diff
@@ -2,6 +2,14 @@
 @import '~@schibstedspain/sui-button-basic/lib/index';
 
 .sui-CoverBasic {
+  @include media-breakpoint-down(s) {
+    height: $hmediabreaks-cover-basic;
```

Comment:
I'll change it (because that name I gave it is awful) but I'm still not sure what is best. I wanted to mean "height value applied only on breakpoint 's'" and I couldn't find a better any better name. On the other hand "h-cover-basic-small" is ambiguous for me because it sounds like the "general height value is 'small'" instead.